jaxrs-tck/pom.xml

```diff
@@ -153,7 +153,11 @@
             <version>2.0.1</version>
         </dependency>
 
-
+        <dependency>
```

Comment:
This dependency should not be a part of the TCK. It is required for the jaxrs-tck/src/main/java/jakarta/ws/rs/tck/ee/rs/ext/interceptor/containerreader/interceptorcontext/Resource.java class, which does not require the bean validation annotations. They can be removed, the TCK does not use them. The BV annotations are likely a leftover from the interoperability tests between JAX-RS and BV, in the platform TCK.

These BV annotations can be removed, but it can be also done by a separate PR.

Reply:
Thanks for the review Jan. I will remove this in a separate PR post completion of migration.